soc/riscv/esp32c3/Kconfig.defconfig

```diff
@@ -8,6 +8,10 @@ if SOC_ESP32C3
 config SOC
 	default "esp32c3"
 
+config SOC_TOOLCHAIN_NAME
+	string
+	default "riscv32-esp-elf"
```

Comment:
You know the arch and you know the soc, so it should be possible to deduce this in similar way as is done for the Zephyr SD here: https://github.com/zephyrproject-rtos/sdk-ng/blob/main/cmake/zephyr/target.cmake

Reply:
Sure, I'll take it.